docs/reference-use-cases.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Cluster API Reference Use Cases
+creation-date: 2019-04-16
+last-updated: 2019-04-16
+---
+
+# Cluster API Reference Use Cases
+
+This is a living document that serves as a reference and a staging area for use cases collected during post-v1alpha1 project redesign.
+
+## Table of Contents
+
+* [Reference Use Cases](#cluster-api-reference-use-cases)
+  * [Metadata](#metadata)
+  * [Table of Contents](#table-of-contents)
+  * [Role Glossary](#role-glossary)
+  * [Icon Glossary](#icon-glossary)
+  * [Operator of Workload Cluster](#operator-of-workload-cluster)
+  * [Operator of Management Cluster](#operator-of-management-cluster)
+  * [Multi-Cluster/Multi-Provider](#multi-cluster/multi-provider)
+  * [Provider Implementors](#provider-implementors)
+
+## Role Glossary
+- __User__: consumer of a Kubernetes-conformant cluster created by the Cluster API.
+    - Does not use Cluster API.
+- __Operator__: Administrator responsible for creating and managing a Kubernetes cluster deployed by Cluster API.
+    - Uses Cluster API.
+- __Multi-cluster operator__: An operator responsible for multiple Kubernetes clusters deployed by Cluster API.
+    - Uses Cluster API.
+    - Cares about keeping config similar between many clusters.
+
+![Cluster API Use Cases](https://user-images.githubusercontent.com/3118335/56234753-d639e980-603a-11e9-8d45-944c547a3280.png)
```

Comment:
Some question for this diagram:
1) How about adding a table to describe `Management Cluster`,  `Control Plane Cluster`, `Workload Cluster`?
2) For the second picture, there is `Workload control plane` running on `Control Plane Cluster`, what do you mean for this? I think the `Workload control plane` is the workload Cluster master node, why it is running on `Control Plane Cluster`?
3) )I found there is no content mentioning if the cluster is HA or not, and I assume the cluster here should support both HA and non-HA cluster, right? How about highlight this in the document?

Reply:
+1 to provide additional info , at least some descriptions to 3 maps including their basic info and some difference..

Reply:
I'm open to remove the diagram, it's meant to be just as a reference, most of this work has been done in https://docs.google.com/document/d/13OQjn5lxRyiW9itNPjVuP0aN_JvYJByKznQESQEDDt8/edit#heading=h.8b9zw0k5lf83 I'm not sure who specifically contributed the image

Reply:
I like the diagram. makes it easier to see some of the use cases. :)